cfg/2.0.0/definitions.yaml

```diff
@@ -5135,48 +5292,8 @@ groups:
             # sysctl -w net.ipv6.conf.default.accept_ra=0
             # sysctl -w net.ipv6.route.flush=1
             
-      scored: true                  
-            
+      scored: true 
 
```

Comment:
Looks like an un-numbered test for "Ensure IPv6 is disabled" is removed here - is that expected?

Reply:
Yes this test got changed into a whole chapter "3.5.1 Configure IPv6 ip6tables"